push-artifacthub/action.yml

```diff
@@ -0,0 +1,24 @@
+name: 'kubewarden-check-artifacthub'
+description: 'Push artifacthub files to artifacthub branch'
+branding:
+  icon: 'package'
+  color: 'blue'
+runs:
+  using: "composite"
+  steps:
+    -
+      name: Checkout code
+      uses: actions/checkout@v3
+    -
+      name: Check that artifacthub-pkg.yml is up-to-date
+      shell: bash
+      run: |
+        git config user.name "Update artifacthub branch"
+        git config user.email github-actions@github.com
+
+        git checkout artifacthub || git checkout --orphan artifacthub
+        git reset HEAD -- .
+        git add artifacthub-pkg.yml artifacthub-repo.yml
+        VERSION=$(sed --posix -n 's,^version: \(.*\),\1,p' artifacthub-pkg.yml)
+        git commit -m "Update Artifact Hub files, version $VERSION"
+        git push --force origin artifacthub
```

Comment:
Using an orphan branch simplifies everything, but is artifacthub going to be fine with it?

Reply:
it seems to be ok with an orphan branch, it finds the `artifacthub-pkg.yml` and the `artifacthub-repo.yml`.

The only problem I see, is missing the README.md. [artifacthub-pkg.yml `readme` field, if missing, gets read from the README.md file](https://github.com/artifacthub/hub/blob/88aae95607ca1905ee93059a55601e4635732678/docs/metadata/artifacthub-pkg.yml#L31).

I will `git add README.md` too.

Reply:
Done in https://github.com/kubewarden/github-actions/pull/69/commits/a53058f1efc5f70dddb67b60d3fb4b7a824ffd9c.